src/common/rivetsExtra.es6

```diff
@@ -60,10 +60,12 @@ rv.formatters['and-not'] = (vlaue, other) => vlaue && !other;
 rv.formatters['gt'] = (vlaue, other) => vlaue > other;
 /* rivets formatter for < operator  */
 rv.formatters['lt'] = (vlaue, other) => vlaue < other;
+/* rivets formatter to check if a variable has value */
+rv.formatters['has-value'] = (value) => value != null;
 /* localise price format*/
-rv.formatters['format-price'] = (value, currency) => (value) ?  formatPrice(value, currency) : undefined;
+rv.formatters['format-price'] = (value, currency) => (value != null) ?  formatPrice(value, currency) : undefined;
```

Comment:
same here

Reply:
I think we can use `isNaN` here, because the function is specific to numbers. What do you think?